test/shared/serializationTests.ts

```diff
@@ -1049,11 +1050,12 @@ describe("msrest", function () {
       deserializedSawshark.siblings.length.should.equal(1);
       deserializedSawshark.siblings[0].fishtype.should.equal("mutatedshark");
```

Comment:
nit: Could we use deep strict equal here to compare two structures?

Reply:
Unfortunately, no as there are some properties that are ignored in the comparison.